installer/config.yml

```diff
@@ -317,6 +317,47 @@ stream:
         # > default is false
         mobile: false
 
+    hls:
+      - # Whether the output is enabled.
+        # > default is false
+        enabled: false
+        # > must be one of ('mpegts', 'mp3', 'adts', 'mp4')
+        # > this field is REQUIRED
+        format: mpegts
+        # > segment_duration  (default:2.0)
+        segment_duration: 2.0
+        # > segments count  (default:5)
+        segment_count: 5
+        # > segments_overhead (default:5)
+        segments_overhead: 5
+        # Output public url, If not defined, the value will be generated from
+        # the [general.public_url] hostname, the output port and mount.
+        public_url:
+        # hls mount point
+        # > this field is REQUIRED
+        mount: hls/main
```

Comment:
The word "mount" is from the icecast/shoutcast vocabulary, I am not sure we want to have if in the context of HLS.

I think this should also include the playlist extension .m3u8

Reply:
ok for m3u8 but i don't know if we need to change the mount property name it's quite coherent for a user perspective...
What would you suggest? manifest?

Reply:
I believe playlist is the word used in the spec.

See https://github.com/libretime/libretime/pull/2234/files#diff-63e70e96336c955e4d36d6178baa323760305a40547cdd6de14ad8e1c64c7c2e

Reply:
Are you okay with these settings?:

```yml
stream:
  outputs:
    # HLS output manifests.
    # > max items is 10
    hls:
      - # Whether the output is enabled.
        # > default is false
        enabled: false
        # Output public url. If not defined, the value will be generated from
        # the [general.public_url] hostname/port.
        public_url:
        # hls playlist name.
        # > this field is REQUIRED
        playlist: "main.m3u8"
        # > segment_duration (default:2.0)
        segment_duration: 2.0
        # > segments count: segments count buffered (default:5)
        segment_count: 5
        # > segments_overhead: segments count kept past the playlist size for those listeners who are still listening on outdated segments. (default:5)
        segments_overhead: 5
        streams:
          - # > prefix of generated segments
            # > this field is REQUIRED
            segments_prefix: mp3_low
            # > format of the container must be one of ('mpegts', 'adts')
            # > this field is REQUIRED
            format: mpegts
            # > codec used for the stream, must be one of ( 'libmp3lame', 'aac')
            # > this field is REQUIRED
            codec: libmp3lame
            # > bitrate of the stream
            # > this field is REQUIRED
            bitrate: 32k
            # > audio sampling rate
            # > this field is REQUIRED
            sample_rate: 44100
          - fragment_prefix: mp3_hifi
            codec: libmp3lame
            bitrate: 256k
            sample_rate: 44100

```